### DISCUSSION TITLE
Notion and Freeplance together

### DISCUSSION THREAD

**Discussion by alecthegeek** [freeplane/freeplane#322] [Show and tell]
I like Freeplane to help me learn, and I like [Notion](https://www.notion.so/) for my notes database.

So this is how I embed Freeplane content in Notion.

It's a little bit clunky but keeps all the info in one place

https://alecthegeek.notion.site/Embed-unsupported-content-in-Notion-e78c086e49de4620a5951a398dbea5cc

YMMV -- open to suggestions on how this could be better, or why I should do it differently.

**Comment by rexelbartolome** [comment]
I like Notion too :D though I've found using Freeplane + Calendar task management much more suitable for my needs. Anyway, I think you can also import an Outline of the map natively written in Notion in addition to the embedded jpeg and source files.

Copy pasting branches and nodes automatically adds the respective indention which I think gets detected by Notion and convert it to a proper list.

For example I could copy these two nodes

![image](https://user-images.githubusercontent.com/88552647/163918903-9b429442-bdef-4779-8bb6-450b036be825.png)

Then I paste it over here in github, this is what comes out. 

edges
    hard edge vs soft edge
    avoid soft edges
tones
    wrong brush size
    inconsistent tones

(what you see is different from what I see in the editor) 
![image](https://user-images.githubusercontent.com/88552647/163919466-4e7ce14f-7aa5-46da-8fd9-e4e0e14a6f7a.png)


You can also experiment with some of the html exports, they're slightly more interactable than a jpeg though you'd need to download and open it again instead of viewing all of it through a Notion page.

**Reply by bepolymathe** [reply]
> I've found using Freeplane + Calendar task management much more suitable for my needs. 

Could you please clarify this point ?
